CHANGES.rst

```diff
@@ -20,8 +20,15 @@ development at the same time, such as 4.5.x and 5.0.
 Unreleased
 ----------
 
-Nothing yet.
+- The semantics of stars in file patterns has been clarified in the docs.  A
+  leading or trailing star component matches any number of path components,
+  like a double star would.  This is different than the behavior of a star in
+  the middle of a pattern.  This discrepancy was `identified by Sviatoslav
+  Sydorenko <starbad_>`_, who `provided patient detailed diagnosis
```

Comment:
On a somewhat unrelated note, I wanted to share one practice I came up with for crediting people in changelogs in projects using Towncrier fragments. I've added a `:user:` RST role to Sphinx (example: https://github.com/pypa/setuptools/blob/1ef36f2/docs/conf.py#L138) and encourage the contributors to add bylines with that role. It expands into a GitHub Sponsors link. When it's not activated, GH makes a redirect to the normal user page.
I borrowed part of the idea from Tox — the only link GH profiles. Here's what it looks like: https://setuptools.pypa.io/en/latest/history.html#v67-0-0.
I use it in some of the projects I maintain or just contribute to.

I figured, if this document is targeted at being consumed by Sphinx and not other tools/systems, you might want to implement something similar in the future.

Reply:
That is a cool idea, I will look into it.  This changelog becomes part of the GitHub releases, so I have to be careful about unusual features.

Reply:
Yeah, that's one pain point. You need to convert to GitHub-Flavoured Markdown for GH Releases. I used pandoc for this in some workflows. Though, since pandoc doesn't know of Sphinx roles (only about the docutils' ones), I had to arm with regexes and pre-process those files via `sed`.